creation/models.py

```diff
@@ -2,27 +2,42 @@
 from django.conf import settings
 from django.contrib.auth.models import User
 
+
 class Language(models.Model):
-    name = models.CharField(max_length = 255, unique = True)
+    name = models.CharField(max_length=255, unique=True)
     user = models.ForeignKey(User)
-    code = models.CharField(max_length = 10, default = 'en')
-    created = models.DateTimeField(auto_now_add = True)
-    updated = models.DateTimeField(auto_now = True)
+    code = models.CharField(max_length=10, default='en')
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
 
     class Meta:
         ordering = ('name',)
 
     def __unicode__(self):
         return self.name
-    
+
+
+class FossSuperCategory(models.Model):
+    category = models.CharField(max_length=255, unique=True)
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        verbose_name = 'FOSS Super Category'
+        ordering = ('category',)
+
+    def __unicode__(self):
+        return self.category
+
 
 class FossCategory(models.Model):
-    foss = models.CharField(unique=True, max_length = 255)
+    foss = models.CharField(unique=True, max_length=255)
     description = models.TextField()
-    status = models.BooleanField(max_length = 2)
+    status = models.BooleanField(max_length=2)
     user = models.ForeignKey(User)
-    created = models.DateTimeField(auto_now_add = True)
-    updated = models.DateTimeField(auto_now = True)
+    category = models.ManyToManyField(FossSuperCategory, null=True)
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
 
     class Meta:
         verbose_name = 'FOSS Categorie'
```

Comment:
You can update `verbose_name='FOSS'` since i don't think `Foss Categorie` make sense here.